### ISSUE TITLE
Curly braces not showing as matrix-like arrangement

### ISSUE THREAD

**Issue by mbumbaugh** [asciimath/asciimathml#101]
Curly braces not showing as matrix-like arrangement. Example:

`{(1,2,3),(4,5,6),(7,8,9)}`

Just shows up on one line like a list. Is that the intention? The syntax seems to indicate it should be a matrix-like arrangement with large, outer curly braces.

**Comment by jipsen** [comment]
Yes, that is the intension, though perhaps unexpected in your case. The documentation could be clearer on this.

{a,b,c} is parsed as a set, and your example is parsed as a set of triples. This overrides the matrix syntax that is otherwise the default. The main reason for this choice is that sets are very common in mathematics and have a simple standard syntax.

To get the expected matrix notation, use invisible brackets and then surround this unparenthesized matrix by {...}: 

{{:(1,2,3),(4,5,6),(7,8,9):}}
